oci/spec_opts_linux.go

```diff
@@ -141,3 +142,11 @@ var WithAllKnownCapabilities = func(ctx context.Context, client Client, c *conta
 func WithoutRunMount(ctx context.Context, client Client, c *containers.Container, s *Spec) error {
 	return WithoutMounts("/run")(ctx, client, c, s)
 }
+
+// WithCDIdevices adds CDI devices to the spec
+func WithCDIdevices(cdiDevices []string) SpecOpts {
```

Comment:
nitpick: shouldn't this be called/spelled `WithCDIDevices`

Reply:
@klihub thank you for the review! renamed.